ui/app/styles/components/stat-text.scss

```diff
@@ -103,15 +103,3 @@
     }
   }
 }
-
```

Comment:
Kept previous use of `columns` and `column` instead for the `<Clients::UsageStat>` template - unless we want to move away from using those?

Reply:
We'll likely replace it with something else once we do move from bulma so that seems fine 👍